include/net/aws_iot.h

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**@file
+ *@brief AWS IoT library header.
+ */
+
+#ifndef AWS_IOT_H__
+#define AWS_IOT_H__
+
+#include <stdio.h>
+#include <net/mqtt.h>
+
+/**
+ * @defgroup aws_iot AWS IoT library
+ * @{
+ * @brief Library to connect the device to the AWS IoT message broker.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief AWS IoT shadow topics, used in messages to specify which shadow
+ *         topic that will be published to.
+ */
+enum aws_iot_topic_type {
+	AWS_IOT_SHADOW_TOPIC_GET = 0x1,
+	AWS_IOT_SHADOW_TOPIC_UPDATE,
+	AWS_IOT_SHADOW_TOPIC_DELETE
+};
+
+/** @brief AWS IoT notification event types, used to signal the application. */
+enum aws_iot_evt_type {
+	/** Connected to AWS IoT broker. */
+	AWS_IOT_EVT_CONNECTED = 0x1,
+	/** Disconnected to AWS IoT broker. */
+	AWS_IOT_EVT_DISCONNECTED,
+	/** Data received from AWS message broker. */
+	AWS_IOT_EVT_DATA_RECEIVED,
+	/** FOTA update done, request to reboot. */
+	AWS_IOT_EVT_FOTA_DONE
+};
+
+/** @brief Struct with data received from AWS IoT broker. */
+struct aws_iot_evt {
+	/** Type of event. */
+	enum aws_iot_evt_type type;
+	/** Pointer to data received from the AWS IoT broker. */
+	char *ptr;
+	/** Length of data. */
+	size_t len;
+};
+
+/** @brief AWS IoT topic data. */
+struct aws_iot_topic_data {
+	/** Type of shadow topic that will be published to. */
+	enum aws_iot_topic_type type;
+	/** Pointer to string of application specific topic. */
+	char *str;
+	/** Length of application specific topic. */
+	size_t len;
+};
+
+/** @brief Structure used to declare a list of application specific topics
+ *         passed in by the application.
+ */
+struct aws_iot_app_topic_data {
+	/** List of application specific topics. */
+	struct mqtt_topic list[CONFIG_AWS_IOT_APP_SUBSCRIPTION_LIST_COUNT];
+	/** Number of entries in topic list. */
+	size_t list_count;
+};
+
+/** @brief AWS IoT transmission data. */
+struct aws_iot_tx_data {
+	/** Topic that the message will be sent to. */
+	struct aws_iot_topic_data topic;
+	/** Pointer to message to be sent to AWS IoT broker. */
+	char *str;
+	/** Length of message. */
+	size_t len;
+	/** Quality of Service of the message. */
+	enum mqtt_qos qos;
+};
+
+/** @brief AWS IoT library asynchronous event handler.
+ *
+ * @param[in] evt The event and any associated parameters.
```

Comment:
`@param` is not aligned with `@brief`, same for the other functions in this file. Personally prefer to keep them aligned, but don't thing code style says anything about it though

Reply:
I agree, it looks better.